### PR TITLE
Try updating repo to v0.14.0-rc3 using Bower

### PR DESCRIPTION
**Pull request by JordanMartinez**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,6 +17,6 @@
   "dependencies": {
     "purescript-datetime": "master",
     "purescript-enums": "master",
-    "purescript-nullable": "master"
+    "purescript-nullable": "main"
```
Comment (thomashoneyman) [inline bower.json:20]: Oh, my.

Comment (thomashoneyman) [reply bower.json:20]: At this point I think we can delete the `master` branch from all those repositories and just have the `main` branch. I _believe_ GitHub redirects 'master' to the default branch automatically (but I may be mistaken -- not able to find a source at the moment).

Comment (JordanMartinez) [reply bower.json:20]: Yeah.....

**Review by thomashoneyman — APPROVED** [review]
👍 once we remove the `--loglevel=debug` statements.

**Pull request merged**